### PR TITLE
Feat/initia publish module

### PR DESCRIPTION
**Pull request by jennieramida**

### CODE REVIEW

File: src/lib/pages/publish-module/index.tsx [hunk 1]
```diff
@@ -1,14 +1,283 @@
...
+      description:
+        "This address can publish these modules again but need to maintain several properties.",
+      condition: true,
+    },
+    {
+      value: "immutable",
+      description: "You cannot publish these modules again with this address",
+      condition: false,
+    },
+  ];
```
Comment (poomthiti) [inline src/lib/pages/publish-module/index.tsx:67]: move outside of component

File: src/lib/pages/publish-module/index.tsx [hunk 2]
```diff
@@ -1,14 +1,283 @@
...
+                />
+              ))}
+            </Flex>
+            <Button
+              onClick={() => append({ module: { file: "", path: "" } })}
+              leftIcon={<CustomIcon name="add-new" />}
+              variant="ghost-primary"
+              p={0}
+              pl={1}
+              pr={1}
```
Comment (poomthiti) [inline src/lib/pages/publish-module/index.tsx:119]: p="0 4px"

**Review by poomthiti — CHANGES_REQUESTED** [review]

File: src/lib/pages/publish-module/components/Footer.tsx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+      bg="gray.900"
+      h="70px"
+      bottom="0"
+      position="sticky"
+      zIndex={2}
+      templateAreas={`"prespace main postspace"`}
+      templateColumns="1fr 5fr 4fr"
+    >
+      <GridItem area="main">
+        <Flex flex="5" align="center" justify="space-between" h="full" w="full">
```
[the remark this answers is not in the document]
Comment (jennieramida) [reply src/lib/pages/publish-module/components/Footer.tsx:26]: false ai poom juke

File: src/lib/pages/publish-module/components/PolicyCard.tsx [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+          <Flex flexDirection="column">
+            <Heading
+              variant="h6"
+              as="h6"
+              fontWeight={600}
+              textTransform="capitalize"
+            >
+              {value}
+            </Heading>
+            <Flex mt={1}>
```
[the remark this answers is not in the document]
Comment (jennieramida) [reply src/lib/pages/publish-module/components/PolicyCard.tsx:33]: ✅

**Pull request merged**